### ISSUE TITLE
Separate HTML Widgets from Mutt Forms

### ISSUE THREAD

**Issue by nicksnell** [mutt-forms/mutt-forms#1]
Remove legacy HTML forms support form mutt-forms core as this is no longer needed by default.

**Comment by andypattenden** [comment]
Would this be moved into a separate plugin for MF?

**Comment by nicksnell** [comment]
Yeah I think so, I don't see any reason not to keep it although haven't seen it used for quite some time
